GameServ/GameHost.cpp

```diff
@@ -484,6 +504,8 @@ void dm_read_sdl(GameHost_Private* host, GameClient_Private* client,
         host->m_localState.add(update);
         host->m_ageSdlHook.add(update);
```

Comment:
This mess is the result of us supporting Global SDL. `m_localState` is the age's state sans Global SDL for saving to the vault while `m_ageSDLHook` is `m_localState` with the global blob merged in.